climate_scanner/trends_innovation_classifier/src/fasttext/modules/trends.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+import os
+import yaml
+from .data_utils import load_params
+
+import glob
+import fasttext
+import operator
+from operator import itemgetter
+
+
+
+# _ROOT = os.path.abspath(os.path.dirname(__file__))
+
+def list_all_classifiers():
+    """
+    A function which lists the available classifiers.
+    :return: list of categories of trends and innovations for which there are classifiers.
+    """
+    
+    params = load_params()
+    inputfilepath = params['data']['path_to_annotated_data']
```

Comment:
No need to read the data filenames rather than the model filenames directly. This causes an error if the two are mismatched, as they are currently